stories/topics/proc-aap-gcp-application-uninstall.adoc

```diff
@@ -0,0 +1,50 @@
+[id="proc-aap-gcp-application-uninstall_{context}"]
+
+= Uninstall Application
+
+[IMPORTANT]
+====
+This action can n ot be reversed.
+====
+
+[IMPORTANT]
+====
+If a Load Balancer was added for the deployment, it needs to be deleted before uninstalling the deployment.
+====
+
+.Procedure
+. Select the main menu from the top left.
+. Select *Deployment Manager*. If you do not see *Deployment Manager*, select *View All Products*.
+. Select *Deployment Manager*.
+. Tick the checkbox in front of the deployment to be deleted.
+. Click on *Delete* in the top bar, this action can not be reversed.
+
+The deployment is being deleted.
+
+To delete the administration secret.
+
+.Procedure
+. Select the main menu from the top left.
+. Select *Security*. If you do not see *Security*, select *View All Products*.
+. Select *Secret Manager*.
+. Filter with the name of the deployment as the secret name format is <DeploymentName>-aap-admin
+. Click the image::more_actions.png[More,10,25] on the line of the deployment.
+. Select *Delete*, the administration password will be displayed. 
+. Type the name of the secret.
+. Click *Delete Secret*
```

Comment:
Click btn:[Delete Secret].

Reply:
This is not working, when I generate the doc I see `Click btn:[Delete Secret]`

Reply:
Is that the only btn:[] that doesn't resolve?

Reply:
No also some `image:...` are not working